x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -235,9 +241,10 @@ function getBaseTemplate(type: string, templateName: string, mappings: Mappings)
     },
     mappings: {
       // To be filled with interesting information about this specific index
+      /*
```

Comment:
Is this intentional? If so, can we delete this and the comment above?

Reply:
+1 on removing this for now. We can add meta data again if we need it later.